seedlet/header.php

```diff
@@ -28,30 +28,34 @@
 				<?php if ( class_exists( 'WooCommerce' ) ) : ?>
 					<button id="woo-open-menu" class="button open">
 						<span class="dropdown-icon open"><?php echo seedlet_get_icon_svg( 'shopping_cart' ); ?> <?php _e( 'Cart', 'seedlet' ); ?></span>
-						<span class="hide-visually expanded-text"><?php esc_html__( 'expanded', 'seedlet' ) ?></span>
+						<span class="hide-visually expanded-text"><?php esc_html__( 'expanded', 'seedlet' ); ?></span>
+					</button>
+					<button id="woo-close-menu" class="button close">
+						<span class="dropdown-icon close"><?php echo esc_html__( 'Close', 'seedlet' ); ?> <?php echo seedlet_get_icon_svg( 'close' ); ?></span>
+						<span class="hide-visually collapsed-text"><?php echo esc_html__( 'collapsed', 'seedlet' ); ?></span>
 					</button>
 				<?php endif; ?>
 				<?php if ( has_nav_menu( 'primary' ) ) : ?>
 					<button id="primary-open-menu" class="button open">
-						<span class="dropdown-icon open"><?php _e( 'Menu', 'seedlet' ); ?> <?php echo seedlet_get_icon_svg( 'menu' ) ?></span>
+						<span class="dropdown-icon open"><?php _e( 'Menu', 'seedlet' ); ?> <?php echo seedlet_get_icon_svg( 'menu' ); ?></span>
 						<span class="hide-visually expanded-text"><?php _e( 'expanded', 'seedlet' ); ?></span>
 					</button>
+					<button id="primary-close-menu" class="button close">
```

Comment:
Moving the close button here has a side effect on the keyboard navigation in both Spearhead and Seedlet: the close button is no longer accessible by tab after you navigate away from it. You can use escape to exit, but I believe the close button also needs to be reachable by keyboard. 

Master | This PR
------- | ------
![Kapture 2020-10-09 at 10 02 30](https://user-images.githubusercontent.com/5375500/95592783-1c48f100-0a17-11eb-8248-c96fac3bc73d.gif) | ![Kapture 2020-10-09 at 10 03 24](https://user-images.githubusercontent.com/5375500/95592703-06d3c700-0a17-11eb-9622-ebbb49103235.gif)

Also in Seedlet, the site link which is not supposed to be reachable is now the first element you reach after tabbing away from the close button. 

We might be able to hack around this in the JS because we're already trapping the focus that way, but I'm not sure that's a great approach.

Reply:
thanks for point this our, I added a commit to fix it.